UnityEngine.AssetBundles/Editor/Tests/ABModelTests.cs

```diff
@@ -45,25 +45,43 @@ public void TearDown()
     [Test]
     public void AddBundlesToUpdate_AddsCorrectBundles_ToUpdateQueue()
     {
+        // Account for existing asset bundles
+        int numBundles = ABModelUtil.BundlesToUpdate.Count;
+
         Model.AddBundlesToUpdate(m_BundleInfo);
-        Assert.AreEqual(3, ABModelUtil.BundlesToUpdate.Count);
+        Assert.AreEqual(numBundles + 3, ABModelUtil.BundlesToUpdate.Count);
     }
 
     [Test]
     public void Update_ReturnsTrue_ForRepaintOnFinalElement()
```

Comment:
I think I misunderstood what this code was doing and trying to test.
Will give it another look and fix.